### ISSUE TITLE
Glasscord support

### ISSUE THREAD

**Issue by sourecorst** [AlexisJonsson/AlexisJonsson.github.io#23]
Is there any plans for adding [glasscord](https://github.com/AryToNeX/Glasscord) support?
It would be quite fitting for this kinda style

**Comment by AlexisJonsson** [comment]
Looking at their github page it seems like it's no longer under development and has security issues. 🤔
So while it looks cool I'm not sure how worth it is to add support for something that has been discontinued. If there's enough people asking for it I might try and do something though.

I'm adding a custom discord background image option, theoretically if you were to pick a transparent png it should work with glasscord.

**Comment by sourecorst** [comment]
Fair enough. I will try to experement with it, myself. If there would be a good results i'll make fork of current repo for it.

**Comment by niruvibes** [comment]
we need this :)
